spec/support/sidekiq.rb

```diff
@@ -6,19 +6,15 @@ def self.logger
   end
 end
 
-# https://github.com/mperham/sidekiq/blob/602d5da96d0101f47d7e89602478b2246853733e/lib/sidekiq/middleware/server/logging.rb
+https://github.com/mperham/sidekiq/blob/aa9745e9b29d408ac423356af50859cb9c382135/lib/sidekiq/job_logger.rb
```

Comment:
This line should not be a comment ? `#`

Reply:
Oh, nice catch :) I accidentally removed this comment symbol before I have committed the changeset.